lib/src/test/scala/TestHelper.scala

```diff
@@ -22,6 +22,7 @@ object TestHelper extends Matchers {
   lazy val apidocApiService: Service = parseFile(s"/examples/apidoc-api.json")
   lazy val dateTimeService: Service = parseFile(s"/examples/date-time-types.json")
   lazy val builtInTypesService: Service = parseFile(s"/examples/built-in-types.json")
+  lazy val scalaKeywordsService: Service = parseFile(s"/examples/response-with-reserved-scala-keyword.json")
```

Comment:
can we add a test to compare the generated output with the expected output?

Reply:
Would Play2ClientGeneratorSpec be the best place for this?